### PR TITLE
Upload files to MVS datasets

### PR DESCRIPTION
**Pull request by kristinochka**
MVS_Upload

If file name is too long, it's automatically truncated.

### CODE REVIEW

**Comment by kristinochka** [comment]
@ChrisBoehmCA Could you take a look?

**Comment by kristinochka** [comment]
Connected to this GitHub issue: https://github.com/zowe/vscode-extension-for-zowe/issues/86

**Comment by kristinochka** [comment]
@Colin-Stone How about changing "Upload" to "Upload member"? Usually the menu specifies the object user is adding, for example  "New File or New Folder". User should understand where they uploading to because they clicked on a dataset.

**Comment by Colin-Stone** [comment]
> @Colin-Stone How about changing "Upload" to "Upload member"? Usually the menu specifies the object user is adding, for example "New File or New Folder". User should understand where they uploading to because they clicked on a dataset.
Works for me.. I would still like the .. to represent a subsequent dialog

**Comment by kristinochka** [comment]
> Upload seems a little ambiguous. Maybe "Upload to.." would be better with the dots indicating that a dialog follows?.
> Seem to have lost the registerCommand check nr12. (zowe.removeSession) I intend to rewrite this piece soon because it is a pain to maintain.

Changed the string to "Upload member..", also added the missing unit test for nr12. I agree, this section needs to be re-factored. Every time a new action is added all other 48 actions need to be updated.

Not sure if I'll have time today to add an integration test. I can also follow up with it in another PR.

File: __tests__/mvs/mvsNodeActions.test.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    return {
+        mSessionNodes: [],
+        mFavorites: [],
+        refresh: mockRefresh
+    };
+});
+
+const testTree = DatasetTree();
+
+describe("mvsNodeActions", async () => {
```
[the remark this answers is not in the document]
Comment (kristinochka) [reply __tests__/mvs/mvsNodeActions.test.ts]: Done.

**Pull request merged**